modules/Schedulers/views/view.detail.php

```diff
@@ -50,7 +53,7 @@ class SchedulersViewDetail extends ViewDetail
     /**
      * @see SugarView::_getModuleTitleListParam()
      */
-    protected function _getModuleTitleListParam()
+    protected function _getModuleTitleListParam($browserTitle = false)
```

Comment:
unused parameter, is something missing here?

Reply:
PHP Strict Standards: Declaration of SchedulersViewDetail::_getModuleTitleListParam() should be compatible with SugarView::_getModuleTitleListParam($browserTitle = false) in \modules\Schedulers\views\view.detail.php on line 66